templates/about/mission.html

```diff
@@ -77,21 +77,21 @@ <h2>SAAS experience, on prem</h2>
 <p>
 Configuration files should not be the way we express our desired state. Configuration files should be the output of something much smarter. A business wants to make high-level, strategic decisions, just as they do with SAAS, and they want to have all of the underlying details handled by someone else, or something else. The Juju operator lifecycle manager, and the Open Operator Collection, deliver on that vision.
 </p>
-<h2>Application management</h2>
+<h2 id="application-management">Application management</h2>
 <p>
 Think of a particular application in your business. It is probably many pieces of software, integrated together, in a scenario. There are probably development, staging, test, and production scenarios that are similar but not exactly the same.
 </p>
 <p>
 The business decisions that are important are:
 </p>
-<ul><li>Where should the application run?
+<ul>
+  <li>Where should the application run?</li>
 
-<li>Which software components should be included in the scenario?
+<li>Which software components should be included in the scenario?</li>
```

Comment:
Indentation is off in this `ul`

Reply:
Oh yea its generated and changes often so indentation for now